src/bin/cql-stress-cassandra-stress/operation/user.rs

```diff
@@ -275,12 +237,15 @@ impl UserOperationFactory {
 
 impl OperationFactory for UserOperationFactory {
     fn create(&self) -> Box<dyn Operation> {
+        // We parsed a user command. This unwrap is safe.
+        let user_profile = self.settings.command_params.user.as_ref().unwrap();
+
```

Comment:
I really dislike this design that requires us to `unwrap` everywhere.

Reply:
I got rid of this by not passing whole `settings` object to the factory. However, we will still discuss the design later offline.